DATABASE.md

```diff
@@ -0,0 +1,40 @@
+# New Database Backup and Import Functionality
+
+**Full activation will take place on approximately July 5, 2025!**
```

Comment:
I dont want to wait a year haha
Can we do 6 months?

Reply:
Time is relative, I can do 6 months